control/lti.py

```diff
@@ -132,9 +132,8 @@ def frequency_response(self, omega=None, squeeze=None):
             outputs=self.output_labels, plot_type='bode')
 
     def dcgain(self):
-        """Return the zero-frequency gain"""
-        raise NotImplementedError("dcgain not implemented for %s objects" %
-                                  str(self.__class__))
+        """Return the zero-frequency (DC) gain."""
+        return NotImplemented
```

Comment:
According to https://docs.python.org/3.12/library/exceptions.html#NotImplementedError , abstract methods should raise an error if they’re not overridden.